frontend/package.json

```diff
@@ -96,7 +96,7 @@
     "lint:lit-analyzer": "lit-analyzer",
     "format": "prettier --write .",
     "format:check": "prettier --check .",
-    "localize:extract": "lit-localize extract",
+    "localize:extract": "lit-localize extract && prettier --write xliff/*.xlf",
```

Comment:
We should remove `.xlf` from `lint-staged` in this case. The pre-commit hook currently runs in this order:
```
    yarn localize:extract
    git add xliff
    npx lint-staged
```

Reply:
Good catch!